.travis.yml

```diff
@@ -42,11 +14,8 @@ rvm:
   - 2.1
   - 2.2
   - ree
+  - rbx-2
 matrix:
-  allow_failures:
-    - env: DB=mysql57
-    - rvm: rbx-2
-    - os: osx
```

Comment:
OS X is a fully supported target; the Travis builds for OS X fail due to Travis issues more often.